flybywiresim/aircraft
# feat(a380x/mfd): add ATCCOM D-ATIS page

Fixes #[issue_no]

## Summary of Changes


To implement D-ATIS page in the ATC COM. This will focus on the layout and some basic UI functionality to prepare for ATSU implementation.

- Implemented the layout for D-ATIS/LIST and D-ATIS/RECEIVED pages.
- Updated IconButton icons to use viewBox property in order to be scalable with the button size. A default size of 35px was retained to maintain compatibility with existing uses of the IconButton.

## Screenshots (if necessary)



![image](https://github.com/user-attachments/assets/66cd1c0d-c79c-4cd9-a621-a5b1be788144)
![image](https://github.com/user-attachments/assets/0066855f-9bcc-45d1-b8db-35a1d5bce2b9)
![image](https://github.com/user-attachments/assets/69761668-aecf-4241-89b0-e4e15542cde8)

## References


![image](https://github.com/user-attachments/assets/c1d53afc-19f8-49b0-bcf1-cbce92b07767)

![image](https://github.com/user-attachments/assets/9f9914fa-be64-47b6-bd63-b7a4bc0220ec)

![image](https://github.com/user-attachments/assets/41087997-0b37-4060-a44b-55f52cc032b1)

https://www.youtube.com/watch?v=AI8yFhuEa4Q&t=5375s




## Additional context



Discord username (if different from GitHub): Heclak

## Testing instructions


1. Navigate to D-ATIS page via ATCCOM menu. Verify that page is showing correctly. Match visuals to screenshots above.
2. Verify the icon buttons on F-PLN page is still rendering correctly.

![image](https://github.com/user-attachments/assets/901cc09e-27c8-4158-a80a-9fd20fab68d7)



## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
